example/cmd/server/main.go

```
@@ -0,0 +1,48 @@
+	"github.com/wearepointers/tycho/example/handlers"
+)
+
+var (
+	dbMigrateFlag = flag.Bool("db:migrate", false, "run database migrations")
+	dbSeedFlag    = flag.Bool("db:seed", false, "run database seeders")
+	dbResetFlag   = flag.Bool("db:reset", false, "reset database migrations")
+)
+
+func main() {
+	flag.Parse()
+	godotenv.Load()
```

golangci-lint (1.62.2)

Error return value of `godotenv.Load` is not checked

(errcheck)